examples/third_party/dat.gui.js

```diff
@@ -573,1958 +561,1950 @@ var inherits = function (subClass, superClass) {
-      _this.setValue(_this.__color.toOriginal());
-      return false;
-    }
-    function setH(e) {
-      if (e.type.indexOf('touch') === -1) {
-        e.preventDefault();
+        var s = (clientX - fieldRect.left) / (fieldRect.right - fieldRect.left);
+        var v = 1 - (clientY - fieldRect.top) / (fieldRect.bottom - fieldRect.top);
+        if (v > 1) {
+          v = 1;
+        } else if (v < 0) {
+          v = 0;
+        }
+        if (s > 1) {
+          s = 1;
+        } else if (s < 0) {
+          s = 0;
+        }
+        _this.__color.v = v;
+        _this.__color.s = s;
+        _this.setValue(_this.__color.toOriginal());
+        return false;
       }
-      var fieldRect = _this.__hue_field.getBoundingClientRect();
-      var _ref2 = e.touches && e.touches[0] || e,
+      function setH(e) {
+        if (e.type.indexOf('touch') === -1) {
+          e.preventDefault();
+        }
+        var fieldRect = _this.__hue_field.getBoundingClientRect();
+        var _ref2 = e.touches && e.touches[0] || e,
           clientY = _ref2.clientY;
-      var h = 1 - (clientY - fieldRect.top) / (fieldRect.bottom - fieldRect.top);
-      if (h > 1) {
-        h = 1;
-      } else if (h < 0) {
-        h = 0;
-      }
-      _this.__color.h = h * 360;
-      _this.setValue(_this.__color.toOriginal());
-      return false;
-    }
-    return _this2;
-  }
-  createClass(ColorController, [{
-    key: 'updateDisplay',
-    value: function updateDisplay() {
-      var i = interpret(this.getValue());
-      if (i !== false) {
-        var mismatch = false;
-        Common.each(Color.COMPONENTS, function (component) {
-          if (!Common.isUndefined(i[component]) && !Common.isUndefined(this.__color.__state[component]) && i[component] !== this.__color.__state[component]) {
-            mismatch = true;
-            return {};
+        var h = 1 - (clientY - fieldRect.top) / (fieldRect.bottom - fieldRect.top);
+        if (h > 1) {
+          h = 1;
+        } else if (h < 0) {
+          h = 0;
+        }
+        _this.__color.h = h * 360;
+        _this.setValue(_this.__color.toOriginal());
+        return false;
+      }
+      return _this2;
+    }
+    createClass(ColorController, [{
+      key: 'updateDisplay',
+      value: function updateDisplay() {
+        var i = interpret(this.getValue());
+        if (i !== false) {
+          var mismatch = false;
+          Common.each(Color.COMPONENTS, function (component) {
+            if (!Common.isUndefined(i[component]) && !Common.isUndefined(this.__color.__state[component]) && i[component] !== this.__color.__state[component]) {
+              mismatch = true;
+              return {};
+            }
+          }, this);
+          if (mismatch) {
+            Common.extend(this.__color.__state, i);
           }
-        }, this);
-        if (mismatch) {
-          Common.extend(this.__color.__state, i);
-        }
-      }
-      Common.extend(this.__temp.__state, this.__color.
```

Comment:
syntax error

Reply:
Fixed in https://github.com/intel/webml-polyfill/pull/525/commits/20dceaa08524973f958fa60704ba07a44e4028ac